gui/src/containers/Configuration/DeleteJobRuns.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import { withStyles } from '@material-ui/core/styles'
+import { useHooks, useState } from 'use-react-hooks'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import PaddedCard from 'components/PaddedCard'
+import Divider from '@material-ui/core/Divider'
+import Typography from '@material-ui/core/Typography'
+import Button from '@material-ui/core/Button'
+import Grid from '@material-ui/core/Grid'
+import Slide from '@material-ui/core/Slide'
+import {
+  deleteCompletedJobRuns,
+  deleteErroredJobRuns
+} from 'actions'
+
+const styles = theme => {
+  return {
+    confirmButton: {
+      backgroundColor: theme.palette.error.main,
+      color: theme.palette.common.white,
+      '&:hover': {
+        backgroundColor: theme.palette.error.dark
+      }
+    },
+    deleteRunsDivider: {
+      marginTop: theme.spacing.unit * 3,
+      marginBottom: theme.spacing.unit * 2
+    }
+  }
+}
+
+const WEEK_MS = (1000 * 60 * 60 * 24 * 7)
+
+const DeleteJobRuns = useHooks(({classes, deleteCompletedJobRuns, deleteErroredJobRuns}) => {
+  const updatedBefore = new Date(Date.now() - WEEK_MS).toISOString()
+  const [showCompletedConfirm, setCompletedConfirm] = useState(false)
+  const [showErroredConfirm, setErroredConfirm] = useState(false)
+
+  const confirmCompletedDelete = () => {
+    deleteCompletedJobRuns(updatedBefore)
+    setCompletedConfirm(false)
+  }
+  const confirmErroredDeleted = () => {
+    deleteErroredJobRuns(updatedBefore)
+    setErroredConfirm(false)
+  }
+
+  return (
+    <PaddedCard>
+      <Typography variant='h5' color='secondary'>
+        Delete Runs
+      </Typography>
+
+      <Typography variant='subtitle1' color='textSecondary'>
+        Reduce your database size by deleting completed and or
+        errored job runs. This action will delete runs a week ago
+        and before.
+      </Typography>
+
+      <Divider className={classes.deleteRunsDivider} />
+
+      <Grid container spacing={16}>
+        <Grid item xs={12}>
+          {!showCompletedConfirm &&
+            <Slide direction='right' in={!showCompletedConfirm} mountOnEnter unmountOnExit>
```

Comment:
Why did we start to use animations? I don't remember seeing animations before here and seeing these two buttons slide in on Config page seems extra to me

Reply:
Our designer wants to use animations throughout the app. I chose to add this animation as it's a simple extra wrapper and the seed was planted when pairing on the style guide.